netebpfext/net_ebpf_ext.c

```diff
@@ -757,37 +772,73 @@ net_ebpf_extension_uninitialize_wfp_components(void)
 {
     size_t index;
     NTSTATUS status;
+    int max_retries = 10;
+    LARGE_INTEGER timeout = {0};
+    timeout.QuadPart = -XMSTO100NS(10000); // 10 seconds
+
+    NET_EBPF_EXT_LOG_ENTRY();
 
     if (_fwp_engine_handle != NULL) {
+        // WFP operations may fail if connections are in the middle of being classified and our WFP filters or callouts
+        // are in use. Prior to this function execution, it is expected that the WFP filter objects were removed,
+        // reducing the risk of this failure to occur. However, the following cleanup functions have retry logic built
+        // in to help ensure that the WFP objects are cleaned up properly.
+
         // Clean up the callouts
```

Comment:
```suggestion
        // Clean up the callouts.
```
If you update this file anyway, might as well fix a nit here.
Ok if you don't though.